client/src/css/lifecoaching.css

```diff
@@ -1,47 +1,65 @@
 .lcview {
-    background-color: #dce9ec;
+  background: rgb(233, 210, 123);
+  background: linear-gradient(
+    0deg,
+    rgba(233, 210, 123, 1) 0%,
+    rgba(193, 124, 124, 1) 100%
+  );
```

Comment:
Could you add "padding-bottom: 20px" here? Im seeing a white bar just above the footer and I think this would get rid of it without messing up other css.

Reply:
you were right, that fixed it